cmd/mayactl/app/command/version_test.go

```diff
@@ -0,0 +1,46 @@
+package command
+
+import (
+	"net/http/httptest"
+	"testing"
+
+	"github.com/openebs/maya/pkg/version"
+
+	utiltesting "k8s.io/client-go/util/testing"
+)
+
+var (
+	response2 = `[{"tag_name": "0.5.4"}]`
+)
+
+func TestCheckLatestVersion(t *testing.T) {
+	cases := []struct {
+		installed string
+		behaviour string
+	}{
+		{"v0.5.4", "pos"},
+		{"v0.5.6", "pos"},
+		{"v0.4.1", "pos"},
+		{"", "neg"},
+		{"0.5.7", "neg"},
+	}
+
+	fakeHandler :=
+		utiltesting.FakeHandler{
+			ResponseBody: string(response2),
+		}
+
+	for i, c := range cases {
+		server := httptest.NewServer(&fakeHandler)
```

Comment:
can you make use use of subtests instead of for loop. There are benefits of using the subtests.